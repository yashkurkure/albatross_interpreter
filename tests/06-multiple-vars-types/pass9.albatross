var JQyG string := "bQ3KzwrfLc3g8";
var CT string := JQyG;
var fS int := 47932<=50770<=!19032>!15415+45037%5551||6996-44774*55867<21159^41504||32039==64857;
var rNjUI10 string := "7qlFCJf0DO2cTHOAu";
var CquF6klLSErM string := "9Mwd";
CquF6klLSErM:=rNjUI10;
fS:=46270-!22992==fS>=47244^fS;
return 54192&fS>=fS*fS*33943&&fS>=44769-fS+fS<>49124<=fS|48088&&fS>=61640^fS<>10547;
rNjUI10:=CquF6klLSErM;
return 36137||14732>=fS==52391<24893|fS&36613|54845>=23560<fS%fS*fS&11602>=fS*fS;
