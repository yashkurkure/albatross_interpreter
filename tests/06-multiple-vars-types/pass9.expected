Variable declared "JQyG" type string
Variable declared "CT" type string
Variable read "JQyG" type string
Variable declared "fS" type int
Variable declared "rNjUI10" type string
Variable declared "CquF6klLSErM" type string
Variable read "rNjUI10" type string
Variable written "CquF6klLSErM" type string
Variable read "fS" type int
Variable read "fS" type int
Variable written "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "CquF6klLSErM" type string
Variable written "rNjUI10" type string
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
Variable read "fS" type int
